source/index.txt

```diff
@@ -95,11 +117,33 @@ What's New
 The :doc:`What's New </whats-new>` section lists new features and
 changes in each version.
 
-Additional Learning
--------------------
+Learn
+------
+
+Visit the Developer Hub and MongoDB University to learn more about the
+MongoDB Java driver.
+
+Developer Hub
+~~~~~~~~~~~~~
+
+The Developer Hub provides tutorials and social engagement for
+developers.
+
+To learn how to use MongoDB features with the Java driver, see the `How
+To's and Articles page <https://www.mongodb.com/developer/learn/?content=Articles&text=java#main>`__.
+
+To learn what fellow developers are asking about, experiencing or their
+interests in regards to the Java driver, see the `forums page
+<https://www.mongodb.com/community/forums/tag/java>`__.
```

Comment:
I like the new intro sentence in this sentence. It identifies two topics "tutorials" and "social engagement" followed by paragraphs that describe each of those. I see a couple potential improvements though:

1) I think we can connect the intro to these two paragraphs by either repeating the same or at least similar terms (tutorials and social engagement). Currently, you have "learn how" for "tutorials" and "learn what..." for social engagement. I think "learn how" might be ok for tutorials, but "learn what" doesn't relate that closely to social engagement. Maybe something like "ask questions and engage in discussions" could work.

2) The phrase "interests in regards to" is too wordy and seems indirect. Is there some alternative wording we can use to eliminate it?